internal/config/log.go

```diff
@@ -18,12 +18,13 @@ var (
 )
 
 func init() {
-	logInit()
+	nodeConfig := NewNodeConfigFromEnv()
+	logInit(nodeConfig)
 }
 
-func logInit() {
+func logInit(nodeConfig NodeConfig) {
```

Comment:
```suggestion
func InitLog(nodeConfig NodeConfig) {
```
It's better to export this function and call it directly after validating the config in `cmd/cartesi-rollups-node/main.go`. Then, the `init()` function from line 20 can be removed.

Reply:
Without the default config package `init()` , that initializes the logs, some tests may fail.  For those, the logs must be initialized at those tests